images.CI/download-customrepo.ps1

```diff
@@ -0,0 +1,14 @@
+param(
+    [String] [Parameter (Mandatory=$true)] $CustomRepoUrl,
+    [String] [Parameter (Mandatory=$true)] $CustomRepoBranch
+)
+
+Write-Host "Clean up default repository"
+Remove-Item -path './*' -Recurse -Force
```

Comment:
I wonder if we can set condition on `checkout` task instead of removing repo

Reply:
AFAIK, `checkout` task do not support stage conditions. 
However, we can avoid removing repo, but it will lead to additional workarounds, so let's just leave it that way, at least for now.